src/transform/class/inheritance/UtilInherits.js

```diff
@@ -1,3 +1,4 @@
+import {find} from 'lodash';
```

Comment:
I'd suggest importing the whole LoDash as `_` like done in other files.

When one sees `_.find()` it's immediately clear that the find method comes from LoDash, but when seeing just `find()` one needs to look around in the file to figure out where this function comes from.